sites/next.skeleton.dev/src/components/docs/FrameworkPicker.astro

```diff
@@ -1,18 +1,6 @@
 ---
 import { Icon } from 'astro-icon/components';
-
-export const frameworks = [
-	{
-		name: 'React',
-		slug: 'react',
-		iconName: 'react',
-	},
-	{
-		name: 'Svelte',
-		slug: 'svelte',
-		iconName: 'svelte',
-	},
-];
+import { frameworks } from 'src/lib/frameworks';
```

Comment:
Done 👍

Reply:
This is my bad for not doing a proper review at my workstation over the weekend, but one follow up front this. I'm not sure `/lib` is the appropriate name for this directory. [Astro doesn't appear to take an opinionated stance on this](https://docs.astro.build/en/basics/project-structure/#directories-and-files). My suggestion is if we only have a single `util.ts` file we move to `/src/utils.ts`. My goal here is to avoid having a bunch of directories with a single file in it. We should only introduce a directory if we have 2+ files that reside in that - or if it's part of Astro's recommended structure. Make sense?